tests/instruments/test_instrument.py

```diff
@@ -22,15 +22,16 @@
 # THE SOFTWARE.
 #
 
+
 import time
 from unittest import mock
 
 import pytest
 
 from pymeasure.units import ureg
 from pymeasure.test import expected_protocol
-from pymeasure.instruments import Instrument
-from pymeasure.instruments.instrument import DynamicProperty
+from pymeasure.instruments import Instrument, Channel
+from pymeasure.instruments.common_base import DynamicProperty
```

Comment:
Could you please migrate the tests to conform to the content common_base.py and channel.py, please?

The tests that test instruments with channels can remain in test_instrument.py (just in case the same question occurs to you)

Reply:
It's one of the todos in the initial post.
I had the question, where to place the channels (with instrument), but wanted to wait for the definitive implementation before returning to the tests.

I do not know, whether the common base will work with properties without an instrument. Let me see.